### ISSUE TITLE
Implement Postgres migrations

### ISSUE THREAD

**Issue by darioAnongba** [ConsenSys/quorum-key-manager#203]
Implement Postgres migrations using: https://github.com/golang-migrate/migrate

Migrations should NOT be part of the codebase (not under `src`). Ideally, they should be written in plain SQL

**Acceptance criteria**:
- Implement migrations to initialize tables from table schema
- Add migrations in docker-compose
- Test migrations in e2e 

Note that migrations are not needed in acceptance tests as we start a fresh QKM so we can directly create tables with the latest schema using go-pg `CreateTable` function. This illustrates the fact that migrations are external to the actual application and should not live in `src`.

**Comment by dolanor** [comment]
I started using it (because I was trying to implement the way go-pg handles migrations.
But since I agree we should use golang-migrate, I tried and I'm happy with it.
As a downside, it is by default compatible with `database/sql` drivers. But as you said, it would be outside of the app codebase.
One question, though. Should the app have some flags to handle migrations at runtime? Or it should be done via the golang-migrate binary in another container/process?

From what we discuss with @ggarri, it seemed we agreed on the fact that integrated into the app was a good idea. Did I miss some info?
